vms/platformvm/warp/README.md

```diff
@@ -86,13 +88,13 @@ Once the validator set of a blockchain is willing to sign an arbitrary message `
 4. Encode the selection of the `N` validators included in the signature in a bitset
 5. Construct the signed message from the aggregate signature, bitset, and original unsigned message
 
-## Verifying / Receiving an Avalanche Warp Message
+## Verifying / Receiving an Avalanche Interchain Message
 
-Avalanche Warp Messages are verified within the context of a specific P-Chain height included in the [ProposerVM](../../proposervm/README.md)'s header. The P-Chain height is provided as context to the underlying VM when verifying the underlying VM's blocks (implemented by the optional interface [WithVerifyContext](../../../snow/engine/snowman/block/block_context_vm.go)).
+Avalanche Interchain Messages are verified within the context of a specific P-Chain height included in the [ProposerVM](../../proposervm/README.md)'s header. The P-Chain height is provided as context to the underlying VM when verifying the underlying VM's blocks (implemented by the optional interface [WithVerifyContext](../../../snow/engine/snowman/block/block_context_vm.go)).
 
 To verify the message, the underlying VM utilizes this `warp` package to perform the following steps:
```

Comment:
Will discuss with @michaelkaplan13 and @StephenButtolph, but imo it should be renamed so that the terminology is consistent throughout our repos. However, if the renaming of this package causes a ton of downstream infra to break, then it is probably not worth the trouble; we can update the data types and variables in the package while keeping the package name as 'warp'. 

If we do, it'd probably be renamed to 'interchain'.

Reply:
Yeah 100%. My current take on this (pretty loosely held) is exactly as @meaghanfitzgerald said: we should try to rename any and all references to "warp" anywhere it's not technically difficult to do so. Unfortunately in a few places (such as the precompile function signatures in the EVM), changing the name would create major incompatibilities that are likely not worth the risk/effort.